scarlet/source.py

```diff
@@ -227,6 +227,36 @@ def init_multicomponent_source(sky_coord, frame, observation, bg_rms, flux_perce
     return seds, morphs
 
 
+class RandomSource(Component):
+    """Sources with uniform random morphology.
+
+    For cases with no well-defined spatial shape, this source initializes
+    a uniform random field and (optionally) matches the SED to match a given
+    observation.
+    """
+    def __init__(self, frame, observation=None, **component_kwargs):
+        """Source intialized with a single pixel
+
+        Parameters
+        ----------
+        frame: `~scarlet.Frame`
+            The frame of the model
+        observation: list of `~scarlet.Observation`
+            Observation to initialize the SED of the source
+        component_kwargs: dict
+            Keyword arguments to pass to the component initialization.
+        """
+        C, Ny, Nx = frame.shape
+        morph = np.random.rand(Ny, Nx)
+
```

Comment:
I usually like to scale random optimisation by the amplitude of the signal, that usually makes convergence slightly faster, but that's more of a familly recipe.

Reply:
Amplitudes are in the SED for us